python/pyspark/streaming/tests.py

```diff
@@ -1047,259 +1046,6 @@ def check_output(n):
         self.ssc.stop(True, True)
 
 
-class KafkaStreamTests(PySparkStreamingTestCase):
```

Comment:
Am I correct that all of this Pyspark Kafka integration is 0.8, not 0.10? that structured streaming is the only option now for Pyspark + Kafka?

Reply:
OK, you or @holdenk or @koeninger might want to skim this change to make sure I didn't delete Pyspark + Structured Streaming + Kafka support inadvertentently. I don't think so, but it's not my area so much.

Reply:
I skimmed and seems fine. Will try to take a look few times more while it's open. (don't block by me)